client/src/app/app.routes.ts

```diff
@@ -3,10 +3,12 @@ import { RouterModule, Routes } from '@angular/router';
 
 import { RegisterComponent } from './register/register.component';
 import { SplashComponent } from './splash/splash.component';
+import { ContactsAdultViewComponent } from './contacts-adult-view/contacts-adult-view.component';
 
 const appRoutes: Routes = [
     { path: '', component: SplashComponent },
     { path: 'register', component: RegisterComponent },
+    { path: 'contacts-adult-view', component: ContactsAdultViewComponent },
```

Comment:
path is the actual url that will take the user to that component, so should probably be 'contacts/view' or such.

Reply:
view-contacts maybe?